test/podspec_test.go

```diff
@@ -39,10 +39,17 @@ var _ = Describe(
 			},
 		}
 
+		volumeMounts := []corev1.VolumeMount{
```

Comment:
This mount will bypass your changes. We add a prefix in volume given in the storage section.
You need to add a volume in storage and then test.